context.go

```diff
@@ -70,7 +70,7 @@ func (c *Context) run(language, code, stdinGlob string) (string, Message) {
 		return "", Message{"error", "no code submitted"}
 	}
 
-	sb := newSandbox(lang, code, stdinGlob, c.options)
+	sb := newSandbox(lang.ExecutionDetails, code, stdinGlob, c.options)
```

Comment:
This might have even worked without this change, via embedded things being accessible seamlessly, but it never hurts to be explicit. :)